### PR TITLE
FRIS meshes

### PR DESCRIPTION
**Pull request by irenavankova**
Meshes with high resolution in the southern Weddell Sea and beneath the Filchner Ronne Ice Shelf (FRIS). The meshes go down to 1, 2, 4, and 8 km beneath FRIS and on the continental shelf in front of it.

### CODE REVIEW

**Review by xylar — APPROVED** [review]
@irenavankova, this looks amazing, just as I expected!

One small thing that you had no way to know about is that the name of the branch will need to change before this moves to E3SM, see: https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/2523172/Branch+Tag+and+Version+name+conventions
I would suggest: `ocn/add-fris-meshes`

Another comment is that it would be great to make the PR to Copass for the code you used to create these meshes at the same time as (or before) the E3SM PR for adding the meshes.

One more thing before this goes to E3SM, we need to put all the mesh data from `files_for_e3sm/assembled_files` from Compass for each of these meshes in `/lcrc/group/e3sm/public_html/` and make sure everyone in the E3SM has write permission to those files and directories.

**Comment by xylar** [comment]
@jonbob, could you have a quick look at this before @irenavankova moves it over to E3SM?

**Comment by jonbob** [comment]
@xylar -- sure, let me take a peek. What is our goal here?

File: cime_config/config_grids.xml [hunk 1]
```diff
@@ -2777,6 +2825,34 @@
...
 
+    <domain name="FRISwISC08to60E3r1">
+      <nx>718054</nx>
+      <ny>1</ny>
+      <file grid="ice|ocn">$DIN_LOC_ROOT/share/domains/domain.ocn.FRISwISC08to60E3r1.230926.nc</file>
+      <desc>FRISwISC08to60E3r1 is a MPAS ice/ocean grid with enhanced resolution of 12km in the Southern Ocean around Antarctica and 4 km beneath the Filchner-Ronne Ice Shelf. The high resolution regions smoothly transition to the background resolution of the standard low resolution 60to30km grid:</desc>
+    </domain>
+
+    <domain name="FRISwISC04to60E3r1">
+      <nx>718054</nx>
```
Comment (jonbob) [inline cime_config/config_grids.xml:2836]: @irenavankova - the domain size should probably be different for each of the meshes

**Comment by jonbob** [comment]
I hate to say it, but I'll need to regenerate the mapping files to match the v3 naming convention. It won't take long, but something we'll have to do

**Comment by xylar** [comment]
> I hate to say it, but I'll need to regenerate the mapping files to match the v3 naming convention. It won't take long, but something we'll have to do

Oh, shoot!  Sorry to create extra work for you.

File: cime_config/config_grids.xml (hunk 1 above)
Comment (irenavankova) [reply cime_config/config_grids.xml:2836]: Thank you, I fixed that and pushed the change.

**Comment by irenavankova** [comment]
Moved to E3SM

**Pull request closed without merging**